libkbfs/data_types.go

```diff
@@ -160,29 +160,30 @@ const (
 // (duplicated) block.
 type BlockRefNonce [8]byte
 
-// zeroBlockRefNonce is a special BlockRefNonce used for the initial
+// ZeroBlockRefNonce is a special BlockRefNonce used for the initial
 // reference to a block.
-var zeroBlockRefNonce = BlockRefNonce([8]byte{0, 0, 0, 0, 0, 0, 0, 0})
+var ZeroBlockRefNonce = BlockRefNonce([8]byte{0, 0, 0, 0, 0, 0, 0, 0})
```

Comment:
Why does this need exporting?  I don't see it used in `mdTool`.  Maybe you'll need to use it in the next PR?